### DISCUSSION TITLE
bug反馈，文件名中存在#号时会报路径不存在错误，疑似文件名传参未处理特殊字符，请测试

### DISCUSSION THREAD

**Discussion by sunnyadams** [AlistGo/alist#899] [Q&A]
bug反馈，文件名中存在#号时会报路径不存在错误，疑似文件名传参未处理特殊字符，请测试
![image](https://user-images.githubusercontent.com/20909045/161474144-bc4092e4-ae1f-40b0-9ba7-fcdac6a3436e.png)
![image](https://user-images.githubusercontent.com/20909045/161474168-2099abb7-1868-49b7-a0bf-2b4dcb1729f7.png)

**Comment by yxh0** [comment]
同样遇到一样的问题，**前端不能正常显示**，但是webdav是正常的

**Reply by anwen-anyi** [reply]
> 同样遇到一样的问题，**前端不能正常显示**，但是webdav是正常的

https://alist-doc.nn.ci/docs/features/limitations#文件夹文件名 
仔细看看文档呗~~~~~
